crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -19,14 +20,13 @@ use crate::{
 };
 
 type PositionItem = [f32; 3];
-type ColorItem = [f32; 4];
 
 #[derive(Resource, Default)]
 pub(crate) struct GizmoStorage<T: GizmoConfigGroup> {
-    pub list_positions: Vec<PositionItem>,
-    pub list_colors: Vec<ColorItem>,
-    pub strip_positions: Vec<PositionItem>,
-    pub strip_colors: Vec<ColorItem>,
+    pub(crate) list_positions: Vec<PositionItem>,
+    pub(crate) list_colors: Vec<LinearRgba>,
+    pub(crate) strip_positions: Vec<PositionItem>,
+    pub(crate) strip_colors: Vec<LinearRgba>,
```

Comment:
As far as I can tell, making these pub(crate) makes this a breaking change.

Reply:
The parent type was already `pub(crate)`: this is just a clarity tweak, not a functional difference.

Reply:
Ah OK, missed that. Thanks!